tools/checkout-keystore

```diff
@@ -57,8 +57,8 @@ filter_sq_stderr() {
     grep -vE '^(Compressed|Encrypted) '
 }
 
-sq decrypt --recipient-key "${recipient_key}" \
-  -o "${cleartext}" "${cryptotext}" \
+sq decrypt --recipient-file "${recipient_key}" \
```

Comment:
Looks like this was here:
https://gitlab.com/sequoia-pgp/sequoia/-/commit/db90178d5b61e3310f979fdaefc66610adf0d33e
https://gitlab.com/sequoia-pgp/sequoia/-/issues/933

Happened a couple of years ago; guess I hadn't updated my `sq` install either. (But I have now, and it has the new names.)